test/app/helpers/fuzzy_time_test.rb

```diff
@@ -48,8 +48,9 @@ def test_less_than_23_hours_ago
   end
 
   def test_a_bit_more_than_23_hours_ago
-    ago = helper.ago(now-(23*60*60 + 29*60))
-    assert_equal link_text(now-(23*60*60 + 29*60), "about 23 hours ago"), ago
+    more_than_23_hours = (23*60*60 + 29*60)
```

Comment:
Surprised rubocop is not complaining about space around operators here.  The style is inconsistent, as well.